src/Attestation/Attestation/help/New-AzAttestation.md

```diff
@@ -192,6 +192,8 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 ### System.String
 
+### System.Collections.Hashtable
```

Comment:
nice catch

Reply:
:-)  Thanks.  Just running the ```UpdateMarkdownHelp``` command with the ```UpdateInputOutput``` parameter.  [Doc link.](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
>If you would like to update the inputs/outputs for a markdown file, please run this cmdlet with the -UpdateInputOutput parameter.